blogs/eks-multi-account-spire/README.md

```diff
@@ -295,7 +294,7 @@ kubectl describe cm spire-bundle -n spire
 ```
 Next, install the spire agent using the provided Helm chart: 
 ```bash
-helm install appmesh-spire-agent eks-multi-account-spire/appmesh-spire-agent \
+helm install appmesh-spire-agent appmesh-spire-agent \
```

Comment:
Same as spire-server. Either add ./ prefix or map to eks-charts repo.

Reply:
Added ./ prefix for clarity.